### DISCUSSION TITLE
Osu Lazer crashes as soon as I focus in it.

### DISCUSSION THREAD

**Discussion by Jasonwanplay** [ppy/osu#26285] [Q&A]
Osu lazer works perfectly fine yesterday. However, when I launched osu lazer today, the client crashes as soon as I focus in it (or full screen) it. The client does not crash if its not full screened.

[1704015894.network.log](https://github.com/ppy/osu/files/13800732/1704015894.network.log)
[1704015894.performance.log](https://github.com/ppy/osu/files/13800733/1704015894.performance.log)
[1704015894.runtime.log](https://github.com/ppy/osu/files/13800734/1704015894.runtime.log)
[1704015894.updater.log](https://github.com/ppy/osu/files/13800735/1704015894.updater.log)

**Comment by bdach** [comment]
Please attach Windows Event Viewer info by performing the following steps:
1. Type `Event viewer` in the Windows search bar (shortcut: <kbd>Windows</kbd>+<kbd>S</kbd>) and press <kbd>Enter</kbd>, or open the *Run...* button (<kbd>Windows</kbd>+<kbd>R</kbd>), type in `eventvwr` and press <kbd>Enter</kbd>.
2. The *Event viewer* window will appear. Expand *Windows Logs* on the left and choose *Application*. It might take a while to load.
3. In the *Actions* pane on the right click *Save All Events As...*, then choose a location to save to.
4. On the *Display Information* popup, select the second option, then the checkbox to show all available languages, then *English (UK)* (US would also do fine).
5. Append the resulting `.evtx` file to this issue.

**Reply by Jasonwanplay** [reply]
How do I append the .evtx file. It doesn't support .evtx. (sorry for the late reply as the issue resolved itself until now again)

**Reply by Jasonwanplay** [reply]
But It seems the issue is TDR issue. However, I got a 4070ti, the problem shouldn't be occuring.

**Reply by bdach** [reply]
> How do I append the .evtx file. It doesn't support .evtx. (sorry for the late reply as the issue resolved itself until now again)

Attach a ZIP instead.

> But It seems the issue is TDR issue

What is "TDR"?

**Reply by Jasonwanplay** [reply]
[a.zip](https://github.com/ppy/osu/files/13945943/a.zip)
I cant provide a precise and detailed explanation about TDR as Im not a computer specialist, you can google it yourself :D.

**Reply by bdach** [reply]
An RTX card should not be "TDR'ing" on the game. Also:

> 🖼️ Initialising "Veldrid" renderer with "OpenGL" surface

If you are able to from within the game please switch to another renderer and see if it helps. Either DirectX 11 or OpenGL (legacy).

**Reply by Jasonwanplay** [reply]
If crash were to occur, Im unable to even interact with the game. The game crashes as soon as I focuses it.

**Reply by bdach** [reply]
In that case please go to `%AppData%/osu`, open `framework.ini` in notepad, and change

```
Renderer = OpenGL
```

to
```
Renderer = Automatic
```
